ios/RNPdftron.m

```diff
@@ -52,6 +52,28 @@ - (dispatch_queue_t)methodQueue
     }
 }
 
+RCT_EXPORT_METHOD(getVersion:(RCTPromiseResolveBlock)resolve
+                 rejecter:(RCTPromiseRejectBlock)reject)
+{
+    @try {
+        resolve([@"PDFNet " stringByAppendingFormat:@"%f", [PTPDFNet GetVersion]]);
```

Comment:
Seems to me this should simply return a double. Why prefix PDFNet?

Reply:
Copied from FL.

https://github.com/PDFTron/pdftron-flutter/blob/master/ios/Classes/PdftronFlutterPlugin.m#L544

Reply:
I don't think either should return a string. But it's not worth changing now.